lib/agent.js

```diff
@@ -308,7 +349,11 @@ Agent.prototype.handleUncaughtExceptions = function (cb) {
 }
 
 Agent.prototype.flush = function (cb) {
-  this._instrumentation.flush(cb)
+  if (this._apmServer) {
+    this._apmServer.flush(cb)
+  } else {
+    process.nextTick(cb)
```

Comment:
Again, log something so we know when `_apmServer` is not set.

Reply:
fixed